exotica_python/src/Exotica.cpp

```diff
@@ -572,49 +587,11 @@ PYBIND11_MODULE(_pyexotica, module)
     scene.def("loadSceneFile", &Scene::loadSceneFile);
     scene.def("getScene", &Scene::getScene);
     scene.def("cleanScene", &Scene::cleanScene);
-
-    // CollisionScene-related functions exposed via Scene - NB: may change in future
-    scene.def("isStateValid", [](Scene* instance) {
-        return instance->getCollisionScene()->isStateValid();
-    });
-    scene.def("isStateValid", [](Scene* instance, bool self, double dist) {
-        return instance->getCollisionScene()->isStateValid(self, dist);
-    });
-    scene.def("updateWorld", [](Scene* instance, moveit_msgs::PlanningSceneWorld& world) {
-        moveit_msgs::PlanningSceneWorldConstPtr myPtr(new moveit_msgs::PlanningSceneWorld(world));
-        instance->getCollisionScene()->updateWorld(myPtr);
-    });
-    scene.def("getCollisionRobotLinks", [](Scene* instance) {
-        return instance->getCollisionScene()->getCollisionRobotLinks();
-    });
-    scene.def("getCollisionWorldLinks", [](Scene* instance) {
-        return instance->getCollisionScene()->getCollisionWorldLinks();
-    });
-    scene.def("getRobotDistance", [](Scene* instance, std::string link, bool self, double safeDist) {
-        double d;
-        Eigen::Vector3d p1, p2, norm, c1, c2;
-        instance->getCollisionScene()->getRobotDistance(link, self, d, p1, p2, norm, c1, c2, safeDist);
-        return py::make_tuple(d, norm);
-    });
-    scene.def("getDistance", [](Scene* instance, std::string o1, std::string o2,
```

Comment:
I think we should bring this one back.

Reply:
Done. The return is a vector of proxies.